### ISSUE TITLE
Cherry-picking Sucks

### ISSUE THREAD

**Issue by jamesrdf** [eclipse/rdf4j-doc#29]
RDF4J project should have two active branches one targeting the the next minor release and the other targeting the next patch release. This will make patch release much less expensive and avoid potential human error involved in cherry-picking commits. Issues should be scheduled for a release prior to merging their PRs. Only once an PR is both approved *and* schedule should it be merged into the corresponding branch (i.e. develop for next minor release and master for next patch release). This pushes scheduling decisions to be made while the PR is open, reducing the confusion during the release process.
